### ISSUE TITLE
Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

### ISSUE THREAD

**Issue by bedmison** [OpenDSA/OpenDSA#615]
This shows up in a bunch of places. The a:link color is defined haiku.css and haiku-sup.css. When used with the admonition tag, which has a light grey background, the contrast ratio is too low. 

Fix:
Make the a:link color darker, and the admonition bg color lighter, or both. 

---
From the WCAG Spec:

The visual presentation of [text](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-text) and [images of text](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-image-of-text) has a [contrast ratio](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-contrast-ratio) of at least 4.5:1, except for the following:

Large Text
[Large-scale](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-large-scale) text and images of large-scale text have a contrast ratio of at least 3:1;

Incidental
Text or images of text that are part of an inactive [user interface component](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-user-interface-component), that are [pure decoration](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-pure-decoration), that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

Logotypes
Text that is part of a logo or brand name has no contrast requirement.

**Comment by bedmison** [comment]
Here is a tool for checking the contrast:
https://webaim.org/resources/contrastchecker/

**Comment by bedmison** [comment]
This issue also applies to the code listings too. We need a new pygments theme that addresses contrast.  The default 'sphinx' pygments theme has some contrast issues too, but the process for creating a new pygments theme is pretty gnarly. See https://github.com/sphinx-doc/sphinx/issues/9105 for some pointers.

**Comment by cashaffer** [comment]
We are starting to work on upgrading Sphinx to a newer version (ideally to the latest version, we will see how things go). Will that affect this problem? That is, has Sphinx worked to improve the contrast in that theme in more recent versions?

**Comment by bedmison** [comment]
Maybe, if the version of pygment baked into the updated version of sphinx has changed the themes to fix the contrast issues.

**Comment by bedmison** [comment]
Fixed and merged in master
